PowerShellForGitHub.psd1

```diff
@@ -19,7 +19,6 @@
         # GitHubConfiguration.ps1) depend on some of the code in Helpers being around at load time.
         'Helpers.ps1',
         'GitHubConfiguration.ps1',
-

```

Comment:
That linefeed is intentional.  It's separating out the the non-alphabetically listed files that all of the other files are dependent on.  Please add it back.